app/code/core/Mage/Adminhtml/Block/Template.php

```diff
@@ -78,6 +85,9 @@ public function isOutputEnabled($moduleName = null)
     protected function _toHtml()
     {
         Mage::dispatchEvent('adminhtml_block_html_before', array('block' => $this));
+        if ($this->_eventPrefix !== 'adminhtml') {
```

Comment:
What's the reasoning behind this?

Reply:
@LeeSaferite Event prefix for this class was `adminhtml` (not `adminhtml_template`) before. This was to prevent that an event with same name gets fired twice. Think this should be changed back.

Existing event

    Mage::dispatchEvent('adminhtml_block_html_before', array('block' => $this));

New one

    Mage::dispatchEvent($this->_eventPrefix . '_block_html_before', array('block' => $this));